src/util/IncognitoBrowser.cpp

```diff
@@ -30,30 +34,31 @@ QString getPrivateSwitch(const QString &browserExecutable)
         {"firefox-esr", "-private-window"},
```

Comment:
Is there a reason why there were 4 firefox entries removed, but this `firefox-esr` is still there? Could it be that removing it was missed?